exchange/csvfeed.go

```diff
@@ -11,16 +11,16 @@ import (
 	"time"
 
 	"github.com/rodrigo-brito/ninjabot/model"
-
 	"github.com/xhit/go-str2duration/v2"
 )
 
 var ErrInsufficientData = errors.New("insufficient data")
 
 type PairFeed struct {
-	Pair      string
-	File      string
-	Timeframe string
+	Pair       string
+	File       string
+	Timeframe  string
+	HeikinAshi bool `default:"false"`
```

Comment:
The tag is not necessary
```suggestion
	HeikinAshi bool
```